Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -477,6 +480,7 @@ extension ActivityStreamPanel: DataObserverDelegate {
     // See ActivityStreamDataObserver for invalidation logic.
     func reloadAll() {
         self.getPocketSites().uponQueue(.main) { _ in
+            // TODO Do not reload if this did not return anything?
             self.collectionView?.reloadData()
```

Comment:
Good point. I have some follow up work here too. We shouldnt show the section title if there are no pocket stories!